drivers/uktty/ns16550/Config.uk

```diff
@@ -1,29 +1,12 @@
 menuconfig LIBUKTTY_NS16550
 	bool "NS16550 / 8250"
-	default n
 	depends on ARCH_ARM_64
+	select LIBUKBOOT
+	select LIBUKOFW
 
 if LIBUKTTY_NS16550
+
 menuconfig LIBUKTTY_NS16550_EARLY_CONSOLE
 	bool "Early console"
```

Comment:
Two questions:
- I think we should provide a help description to this option (describing how it works and how the device is detected via fdt/kernel cmdline)?
- Can we introduce the `lib/ukboot` dependency only when this option is enabled. Otherwise we don't need `lib/ukboot` for the driver, correct?

Reply:
> I think we should provide a help description to this option (describing how it works and how the device is detected via fdt/kernel cmdline)?

Same as above, I would treat the fdt requirement in the same scope as the rest of the drivers (i.e. no need to state in the `help` of every driver that it is configured via the fdt, but instead document somewhere globally that arm64 systems are configured via the fdt).

Ditto for the cmdline: Currently the only way to figure out the available parameters is to pass `help`. As we move forward, these could also be documented in each library's `README.md`.

> Can we introduce the lib/ukboot dependency only when this option is enabled. Otherwise we don't need lib/ukboot for the driver, correct?

That's right 👍🏼

Reply:
> Same as above, I would treat the fdt requirement in the same scope as the rest of the drivers (i.e. no need to state in the help of every driver that it is configured via the fdt, but instead document somewhere globally that arm64 systems are configured via the fdt).

What I meant is that the help says which fdt node is checked.

Reply:
Done